### ISSUE TITLE
modTemplate is deprecated since version 3.0

### ISSUE THREAD

**Issue by Ibochkarev** [modxcms/revolution#15471]
### Summary
Error `modTemplate is deprecated since version 3.0`

### Step to reproduce
Set log_level to 4.

Go to the elements tab and open any template

Open MODX error log.

It should be clear.

### Observed behavior
Errors:

```bash
[2021-03-05 15:31:21] (ERROR in xPDO\xPDO::call @ /home/s22270/www/core/vendor/xpdo/xpdo/src/xPDO/xPDO.php : 773) modTemplate is deprecated since version 3.0. Replace references to class modTemplate with MODX\Revolution\modTemplate to take advantage of PSR-4 autoloading.
[2021-03-05 15:31:21] (ERROR in MODX\Revolution\modAccessibleObject::load @ /home/s22270/www/core/src/Revolution/modAccessibleObject.php : 106) modTemplate is deprecated since version 3.0. Replace references to class modTemplate with MODX\Revolution\modTemplate to take advantage of PSR-4 autoloading.
```

A similar error appears in other elements as well.

### Related issue(s)/PR(s)
NA

### Environment
MODX Revolution 3.0.0-alpha3 (git), PHP Version 7.4.12

**Comment by JoshuaLuckers** [comment]
Im working on this!

**Comment by Ibochkarev** [comment]
Fixed #15477
